python/cugraph/traversal/bfs.py

```diff
@@ -15,7 +15,7 @@
 from cugraph.structure.graph import Graph
 
 
-def bfs(G, start):
+def bfs(G, start, return_sp_counter=False):
```

Comment:
Should we split it into 2 functions?
`bfs(G, start)`: never return the sp_counter
`bfs_spc(G, start)`: that always return the sp_counter

Reply:
I am fine with the current, single-function version. People can still use it exactly like before if they want to. Just need to confirm that perf is not impacted in this case.

Reply:
Based on the value of return_sp_counter the C++ is provided with a valid or a null pointer. Based on the value of the pointer shortest path counting specific actions are enable(valid) or disabled(nullptr). Performance should not be affected leaving the return_sp_counter as a parameter of the python function.